selfhost/main.jakt

```diff
@@ -480,9 +480,9 @@ function main(args: [String]) {
                     continue
                 }
 
-                for function_ in scope.functions.iterator() {
-                    if function_.0 == "main" {
-                        main_function_id = function_.1
+                for (function_name, function_ids) in scope.functions.iterator() {
```

Comment:
again, confusing that its called function when its really an overload set....not sure about a good name. overload set? function family? is this bikeshedding?

Reply:
> is this bikeshedding?
Nah, the current name is moot. I don't know if any canonical name exists but [CPP Reference](https://en.cppreference.com/w/cpp/language/overload_resolution) talks about sets of candidate functions, so I think overload set accurately describes the intent. I would still name the member of the Scope `functions` but this is then a mapping of names -> overload sets

Reply:
ok, overload_set already reads a lot nicer 👍